pycbc/strain/gate.py

```diff
@@ -168,16 +168,16 @@ def gate_and_paint(data, lindex, rindex, invpsd, copy=True):
     # Copy the data and zero inside the hole
     if copy:
         data = data.copy()
-    data[lindex:rindex] = 0
+    data[lindex:rindex+1] = 0
```

Comment:
As we discussed on the call, remove the +1 to stay consistent with PyCBC's convention of using right-open intervals. Add a comment about the ambiguity of the gate start though.